packages/date/src/getOptions.ts

```diff
@@ -124,20 +124,19 @@ function getOptions() {
   const localize = {
     ...originalLocale.localize,
 
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- Required workaround
```

Comment:
The _unsafe assignment_ refers to the argument passed to the function. I could update the argument type in the module definition so this passes, but it doesn't seem worth it, as it won't actually represent the true underlying function anyway?

Reply:
Hm, looks like eslint claims this rule was never reported though?
`eslint-comments/no-unused-disable] '@typescript-eslint/no-unsafe-assignment' rule is disabled but never reported.`

Reply:
I'll probably fix the mock type so we can avoid this completely.

Reply:
Fixed in b07750287e.